nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/blob/BlobManager.java

```diff
@@ -67,8 +67,23 @@
      * @param id the blob provider id or namespace
      * @return the blob provider
      * @since 10.10
+     * @deprecated since 11.1, use {@link #getBlobProviderWithNamespace(String, String)} instead
      */
-    BlobProvider getBlobProviderWithNamespace(String id);
+    @Deprecated
+    default BlobProvider getBlobProviderWithNamespace(String id) {
+        return getBlobProviderWithNamespace(id, "default");
```

Comment:
Why not using (moving) the `DEFAULT_ID` constant?

Reply:
Well if we define a constant it would be only used by the deprecated method... I'd rather leave the constants about defaults in whatever is using the API (`KeyValueBlobTransientStore.CONFIG_DEFAULT_BLOB_PROVIDER_DEFAULT`).

Reply:
Makes sense. So maybe we can deprecate `BlobManagerComponent#DEFAULT_ID` 😬